heat/files/newton/heat.conf.Debian

```diff
@@ -30,7 +30,7 @@ verbose = true
 
 # List of directories to search for plug-ins. (list value)
 {%- if server.get('dir', {}).plugins is defined %}
-plugin_dirs={{ server.dir.plugins|join(",") }}
+plugin_dirs=/usr/lib64/heat,/usr/lib/heat,/usr/local/lib/heat,/usr/local/lib64/heat,{{ server.dir.plugins|join(",") }}
```

Comment:
If server.dir.plugins is empty, you will end up with trailing `,`
which is not what you probably want